plugins/polio/js/src/domains/Calendar/campaignCalendar/Filters.tsx

```diff
@@ -78,16 +93,20 @@ const Filters = ({
         if (filtersUpdated) {
             setFiltersUpdated(false);
             const urlParams = {
-                countries,
+                countries: countries?.join(','),
                 search: search && search !== '' ? search : undefined,
-                roundStartFrom: dateRangePickerToDateApi(roundStartFrom),
-                roundStartTo: dateRangePickerToDateApi(roundStartTo),
+                roundStartFrom: dateRangePickerToDateApi(
+                    roundStartFrom ?? undefined,
+                ),
+                roundStartTo: dateRangePickerToDateApi(
+                    roundStartTo ?? undefined,
+                ),
                 page: null,
                 campaignType,
-                showOnlyDeleted: showOnlyDeleted || undefined,
+                showOnlyDeleted: showOnlyDeleted ? 'true' : undefined,
```

Comment:
```suggestion
                showOnlyDeleted: showOnlyDeleted && showOnlyDeleted !== "false" ? 'true' : undefined,
```

Just to avoid `/dashboard/polio/list/accountId/1/showOnlyDeleted/false/campaignType/regular/filterLaunched/true` to show deleted campaigns

Reply:
This comparison appears to be unintentional because the types 'boolean' and 'string' have no overlap.